flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/table/RowDataDebeziumDeserializeSchema.java

```diff
@@ -480,6 +517,39 @@ public Object convert(Object dbzObj, Schema schema) {
         };
     }
 
+    private DeserializationRuntimeConverter createArrayConverter(ArrayType arrayType) {
+        final Class<?> elementClass =
+                LogicalTypeUtils.toInternalConversionClass(arrayType.getElementType());
+        final DeserializationRuntimeConverter elementConverter =
+                createConverter(arrayType.getElementType());
+        return (dbzObj, schema) -> {
+            if (dbzObj instanceof List) {
+                List<?> elements = (List<?>) dbzObj;
+                final Object[] elementArray =
+                        (Object[]) Array.newInstance(elementClass, elements.size());
+                for (int i = 0; i < elements.size(); i++) {
+                    elementArray[i] = elementConverter.convert(elements.get(i), schema);
+                }
+                return new GenericArrayData(elementArray);
+            } else if (dbzObj instanceof String) {
+                // for SET datatype in mysql, debezium will always return a string split by comma
+                // like "a,b,c"
+                String[] elements = ((String) dbzObj).split(",");
+                final Object[] elementArray =
+                        (Object[]) Array.newInstance(elementClass, elements.length);
+                for (int i = 0; i < elements.length; i++) {
+                    elementArray[i] = elementConverter.convert(elements[i], schema);
+                }
+                return new GenericArrayData(elementArray);
+            } else {
+                throw new IllegalArgumentException(
+                        String.format(
+                                "Unable to convert to arrayType from unexpected value '%s', only List and String type can be converted to arrayType",
```

Comment:
```suggestion
                                "Unable convert to Flink ARRAY type from unexpected value '%s', only MySQL LIST and SET type could convert to ARRAY type",
```

Reply:
There's no LIST type in MySQL.
For MySQL SET type, debezium will always return a string. 
And I also consider List type in here is  to make it can be used in PG.